Stronger/Resources/Localizable.xcstrings

```diff
@@ -218,8 +234,39 @@
         }
       }
     },
+    "Goal Resistance" : {
+
+    },
+    "Goal Weight" : {
+
+    },
+    "GOAL_RESISTANCE" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Goal Resistance"
+          }
+        }
+      }
+    },
+    "GOAL_WEIGHT" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Goal Weight"
+          }
+        }
+      }
```

Comment:
Please try to be consistent with the  localization usage, sometimes you use UPPERCASE string and sometimes just normal strings. Either is fine, I would slightly prefer human readable strings for shorter text. But in the end either is fine, just be consistent.